JobConfig/primary/IPAStopParticle.fcl

```diff
@@ -0,0 +1,52 @@
+#
```

Comment:
This is almost identical to TargetStopParticle. I suggest to create StopParticle.fcl with the common lines, and #include that in both TargetStopParticle and IPAStopParticle, then add the few lines that are different.

Reply:
Ok, I created `StopParticle.fcl`, I think it should be fine now